tests/providers/google/cloud/hooks/test_cloud_run.py

```diff
@@ -294,3 +301,73 @@ def mock_get_operation(self, expected_operation):
 
     def _dummy_get_credentials(self):
         pass
+
+
+@pytest.mark.db_test
+class TestCloudRunServiceHook:
+    def dummy_get_credentials(self):
+        pass
+
+    @pytest.fixture
```

Comment:
I think we need more tests here for delete method in hook

Reply:
done.